infra.bs

```diff
@@ -119,6 +119,29 @@ will make the caller automatically rethrow the given value, if any, and thereby
 caller's algorithm. Using prose the caller has the ability to "catch" the exception and perform
 another action.
 
+<h4 id=algorithm-iteration>Iteration</h4>
+
+<p>There's a variety of ways to repeat a set of steps until a condition is reached. The Infra
+Standard is not (yet) exhaustive on this, but here are some examples:
+
+<ul>
+ <li><p>For each, as demonstrated later in this section.
+
+ <li>
+  <p>While
```

Comment:
Oh, and if we're doing to make it a definition, we need to actually define it. E.g. as https://en.wikipedia.org/wiki/While_loop does.